packages/fxa-settings/src/lib/glean/index.test.ts

```diff
@@ -377,6 +377,22 @@ describe('lib/glean', () => {
           }, 20)
         );
       });
+
+      it('submits a ping with the reg_change_email_link_click event name', async () => {
+        GleanMetrics.registration.changeEmail();
```

Comment:
Did you want to add the `isDone` function similar to content-server?

Reply:
> Did you want to add the `isDone` function similar to content-server?

I added the function in a recent PR.  Just haven't gone through and update all the tests to use it.  I can update the tests in this PR.  👍

Reply:
Oh wait I had that backwards... it's the tests in Settings that need updating.